Dockerfile

```diff
@@ -320,6 +321,3 @@ RUN mkdir -p ~/.cargo && printf "[net]\ngit-fetch-with-cli = true\n" >> ~/.cargo
 # Disable automatic pulling of files stored with Git LFS
 # This avoids downloading large files not necessary for the dependabot scripts
 ENV GIT_LFS_SKIP_SMUDGE=1
-
-# Pin to an earlier version of Hex. This must be run as dependabot
```

Comment:
> This must be run as dependabot

This is the other reason I opted to install hex within the helper builder script and not up when we install Erlang / Elixir, as they're installed as the root user.